modules/Finance/invoices_payOnline.php

```diff
@@ -76,22 +73,17 @@
                     $feeTotal += $rowFees['fee'];
                 }
 
-                $currency = getSettingByScope($connection2, 'System', 'currency');
-                $enablePayments = getSettingByScope($connection2, 'System', 'enablePayments');
-                $paypalAPIUsername = getSettingByScope($connection2, 'System', 'paypalAPIUsername');
-                $paypalAPIPassword = getSettingByScope($connection2, 'System', 'paypalAPIPassword');
-                $paypalAPISignature = getSettingByScope($connection2, 'System', 'paypalAPISignature');
-
-                if ($enablePayments == 'Y' and $paypalAPIUsername != '' and $paypalAPIPassword != '' and $paypalAPISignature != '' and $feeTotal > 0) {
+                if ($payment->isEnabled() and $feeTotal > 0) {
                     $financeOnlinePaymentEnabled = getSettingByScope($connection2, 'Finance', 'financeOnlinePaymentEnabled');
                     $financeOnlinePaymentThreshold = getSettingByScope($connection2, 'Finance', 'financeOnlinePaymentThreshold');
+                    $paymentGateway = getSettingByScope($connection2, 'System', 'paymentGateway');
                     if ($financeOnlinePaymentEnabled == 'Y') {
                         echo "<h3 style='margin-top: 40px'>";
                         echo __('Online Payment');
                         echo '</h3>';
                         echo '<p>';
                         if ($financeOnlinePaymentThreshold == '' or $financeOnlinePaymentThreshold >= $feeTotal) {
-                            echo sprintf(__('Payment can be made by credit card, using our secure PayPal payment gateway. When you press Pay Online Now, you will be directed to PayPal in order to make payment. During this process we do not see or store your credit card details. Once the transaction is complete you will be returned to %1$s.'), $session->get('systemName')).' ';
+                            echo sprintf(__('Payment can be made by credit card, using our secure %2$s payment gateway. When you press Pay Online Now, you will be directed to %2$s in order to make payment. During this process we do not see or store your credit card details. Once the transaction is complete you will be returned to %1$s.'), $session->get('systemName'), $paymentGateway).' ';
```

Comment:
Have such strings been updated in POEditor?

Reply:
They have not. I looked at doing this manually, but since the strings are old enough, there's a decent number of translations already in there, and for any translations using different character sets or rtl, it's not just a matter of removing/replacing the word PayPay. We may just have to re-translate these strings 😬